build_msvc/README.md

```diff
@@ -17,16 +17,17 @@ Options for installing the dependencies in a Visual Studio compatible manner are
 - Download the source code, build each dependency, add the required include paths, link libraries and binary tools to the Visual Studio project files.
 - Use [nuget](https://www.nuget.org/) packages with the understanding that any binary files have been compiled by an untrusted third party.
 
-The external dependencies required for the Visual Studio build are (see the [dependencies doc](https://github.com/bitcoin/bitcoin/blob/master/doc/dependencies.md) for versions):
+The external dependencies required for the Visual Studio build are (see [dependencies.md](https://github.com/bitcoin/bitcoin/blob/master/doc/dependencies.md) for more info):
 
-- Berkeley DB,
-- OpenSSL,
-- Boost,
-- libevent,
+- Berkeley DB
+- OpenSSL
+- Boost
+- libevent
 - ZeroMQ
+- RapidCheck
```

Comment:
This is not "required for the Visual Studio build", it is a test-only optional experimental package

Reply:
@MarcoFalke I cannot compile using Visual Studio unless it is installed.

Reply:
That sounds like a bug in our build_msvc setup

Reply:
@MarcoFalke the rapidcheck header is used in two of the test classes: src/test/gen/crypto_gen.cpp and src/test/key_properties.cpp.

Reply:
Yeah, but shouldn't it be guarded by a configure check. Those probably don't exist on windows, but I wanted to mention it.

Reply:
The checks in the key_properties.cpp class get hit when I run the test_bitcoin.exe app under Windows. From a quick look at the RapidCheck library there doesn't seem to be any reason it can't be used on Windows?

![rc_breakpoint](https://user-images.githubusercontent.com/197660/59793958-18e89e00-92d8-11e9-8ac5-2713afc2d123.png)

Reply:
Oh, I am pretty sure it can be used on Windows, but maybe not every developer *wants* to use it on windows or even install it?

Reply:
I'd recommend adding `rapidcheck` to the build instructions for msbuild/Visual Studio as per this PR. It took 24s to install on my Windows machine. I don't think it's going to cause problems for any devs.

```
c:\Tools\vcpkg>vcpkg install rapidcheck:x64-windows-static
The following packages will be built and installed:
    rapidcheck[core]:x64-windows-static
Starting package 1/1: rapidcheck:x64-windows-static
Building package rapidcheck[core]:x64-windows-static...
-- Using cached C:/Tools/vcpkg/downloads/emil-e-rapidcheck-cf9e0d8bd8c94e9dc00dc0ab302352bfaf1a3ac5.tar.gz
-- Using source at C:/Tools/vcpkg/buildtrees/rapidcheck/src/bfaf1a3ac5-d497ee5770
-- Configuring x64-windows-static
-- Building x64-windows-static-dbg
-- Building x64-windows-static-rel
-- Performing post-build validation
-- Performing post-build validation done
Building package rapidcheck[core]:x64-windows-static... done
Installing package rapidcheck[core]:x64-windows-static...
Installing package rapidcheck[core]:x64-windows-static... done
Elapsed time for package rapidcheck:x64-windows-static: 23.58 s

Total elapsed time: 23.58 s

The package rapidcheck:x64-windows-static provides CMake targets:

    find_package(rapidcheck CONFIG REQUIRED)
    # Note: 2 target(s) were omitted.
    target_link_libraries(main PRIVATE rapidcheck rapidcheck_boost rapidcheck_catch rapidcheck_gmock)


c:\Tools\vcpkg>
```